pytorchvideo/data/labeled_video_paths.py

```diff
@@ -67,6 +70,59 @@ def from_csv(cls, file_path: str) -> LabeledVideoPaths:
         ), f"Failed to load dataset from {file_path}."
         return cls(video_paths_and_label)
 
+
+    @classmethod
+    def from_df(cls, df:pd.DataFrame) -> LabeledVideoPaths:
+        """
+        Factory function that creates a LabeledVideoPaths object by reading a dataframe.
+        Sample dataframe
+        df=pd.DataFrame(
+            {
+                "path":["path_to_video_1","path_to_video_2","path_to_video_3"],
+                "label":["label_1","label_2","label_3"]
+            })
+
+        Args:
+            df (dataframe): The dataframe variable.
+        """
+        video_paths_and_label = []
+        for row in df.iterrows():
+            path,label=row[1].values
+            video_paths_and_label.append((path, int(label)))
+
+        assert (
+            len(video_paths_and_label) > 0
+        ), f"Failed to load dataset from df."
+        return cls(video_paths_and_label)
+
+
+
+
+
+    @classmethod
+    def from_df(cls, df: pd.DataFrame) -> LabeledVideoPaths:
+        """
+        Factory function that creates a LabeledVideoPaths object by reading a dataframe.
+        Sample dataframe
+        df=pd.DataFrame(
+            {
+                "path":["path_to_video_1","path_to_video_2","path_to_video_3"],
+                "label":["label_1","label_2","label_3"]
+            })
+
+        Args:
+            df (dataframe): The dataframe variable.
+        """
+        video_paths_and_label = []
+        for row in df.iterrows():
+            row = row[1].values
+            path = row[0]
+            label = row[1::].astype(float)
+            video_paths_and_label.append((path, label))
+
+        assert len(video_paths_and_label) > 0, f"Failed to load dataset from df."
+        return cls(video_paths_and_label)
+
```

Comment:
It looks like some duplicated code here. Which one are you using? If the second one. It looks fine. You may be able to optimize it slightly more, but it's nothing show-stopping.

Reply:
Had it from the original PR i mentioned which was in original pytorchvideo. I'll remove 1 func as it is duplicate.

Reply:
Merging as removed